organization/governance-model.md

```diff
@@ -32,34 +34,41 @@ The general assembly:
 * approves the Foundation's operational plan for the coming year
 * elects the board of directors
 
-These decisions are agreed through a majority vote. Voting rights are proportional to members' membership dues. This ensures democratic ownership and control by all members.
+These decisions are agreed through a majority vote.
+Each member has 3 votes.
```

Comment:
This is perhaps over-simplifying it. We wanted the representatives to act individually, but this implies that they act in unison. (But also, perhaps that level of detail is not vital here, but it is enough that it is spelled out in the bylaws.)

Reply:
@clausmullie and I envisioned this as an overview of the governance for a brand new staff member. In that context, would you appreciate that level of detail?

Reply:
I think this is fine enough in that regard.